### ISSUE TITLE
Add task argument for CLI interface

### ISSUE THREAD

**Issue by ChristopherMancuso** [krishnanlab/PyGenePlexus#142]
I think it would be good to have a user be able to use the CLI interface even if they don't want to do the similarity to other models to generate a network. For instance, if someone adds a new network or GSC, they no longer can use the CLI interface

**Comment by ChristopherMancuso** [comment]
That takes care of the problem I was thinking about. I didn't see that argument was included. Is there a reason that argument is not on the Read the Docs (https://pygeneplexus.readthedocs.io/en/main/notes/cli.html)?

**Comment by RemyLau** [comment]
@ChristopherMancuso good point, it seems like it's not even initialized in the parse_args function. I'll fix that

**Comment by RemyLau** [comment]
Actually it is initialized. Only need to update doc

https://github.com/krishnanlab/PyGenePlexus/blob/02898907374a73969539b26c60efa4687a13bbe2/geneplexus/cli.py#L139-L145
